<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>client</class>
 <widget class="QDialog" name="client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1900</width>
    <height>980</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
	
	
QWidget {	
	border:1px solid #dfdfdf;
	border-radius: 5px;
	background-color: #ffffff;
}

QPushButton{
	
	border-raduis:5px
}
QPushButton:pressed{
	
	
	background-color: rgb(185, 185, 185);
}
#side_menu{
	border-radius: 20px;
	/*background-color: #071e26;*/
	color: rgb(0, 0, 0);
}
#side_menu_2{
	border-radius: 20px;
	/*background-color: #071e26;*/
	background-color:rgb(0,0,0);
}

#top_widget{

	/*background-color: #071e26;*/
background-color:rgb(0,0,0);
}
#widget_2 #widget{
	border-radius: 20px;
	background-color: rgb(0,0,0);
}

#form{
	
	
	background-color: rgb(255,255,255);
border-top-left-radius: 50px;
}

#widget_2 QLineEdit{
font: 11pt &quot;MS Shell Dlg 2&quot;;
background-color:rgba(0,0,0,0);
border:none;
border-bottom:2px solid rgb(0,0,0);
padding-bottom:7px

}
#widget_2{
	
	/* background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #ffa02f, stop: 1 #d7801a);*/
	/*background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #4d4d4d, stop: 0.1 #646464, stop: 1 #5d5d5d);*/
    /*background: QLinearGradient( x1: 0, y1: 0, x2: 1, y2: 0, stop: 0 #ffa02f, stop: 1 #d7801a);*/
/*background-color: linear-gradient(135.9deg, rgb(109, 25, 252) 16.4%, rgb(125, 31, 165) 56.1%);
background: QLinearGradient( x1: 0, y1: 0, x2: 1, y2: 0, stop: 0 #6d19fc, stop: 1 #7d1fa5);*/
	background-color:QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #4aafdb, stop: 1 #090763);
}

#widget_2 #ajouter{
    color:rgb(0, 0, 0);
   
	background-color: rgb(172, 172, 172);
    border-width: 1px;
    border-color: #1e1e1e;
    border-style: solid;
    border-radius: 30px;
    padding: 3px;
    font-size: 12px;
    padding-left: 5px;
    padding-right: 5px;	
}

#widget_2 #modifier{
    color:rgb(0, 0, 0);
   
	background-color: rgb(172, 172, 172);
    border-width: 1px;
    border-color: #1e1e1e;
    border-style: solid;
    border-radius: 30px;
    padding: 3px;
    font-size: 12px;
    padding-left: 5px;
    padding-right: 5px;	
}

#widget_2 #supprimer{
    color:rgb(0, 0, 0);
   
	background-color: rgb(172, 172, 172);
    border-width: 1px;
    border-color: #1e1e1e;
    border-style: solid;
    border-radius: 30px;
    padding: 3px;
    font-size: 12px;
    padding-left: 5px;
    padding-right: 5px;	
}

#options{
/*background-color: #071e26;*/
background-color: rgb(0, 0, 0);
border-bottom-right-radius: 50px;

}
#widget_2 #recherche_2{
	
	background-color: rgb(204, 204, 204);
	border:1px solid rgb(0,0,0);
}
#widget_2 #recherche_3{
	
	background-color: rgb(204, 204, 204);
	border:1px solid rgb(0,0,0);
}
#widget_2 #recherche_4{
	
	background-color: rgb(204, 204, 204);
	border:1px solid rgb(0,0,0);
}


/*#widget_2 #ajouter{
 color: #b1b1b1;
    background-color:rgb(163, 163, 163)
    border-width: 1px;
    border-color: #1e1e1e;
    border-style: solid;
    border-radius: 30px;
    padding: 3px;
    font-size: 12px;
    padding-left: 5px;
    padding-right: 5px;
}


#modifier{
 color: #b1b1b1;
    background-color: #565656;
    border-width: 1px;
    border-color: #1e1e1e;
    border-style: solid;
    border-radius: 30px;
    padding: 3px;
    font-size: 12px;
    padding-left: 5px;
    padding-right: 5px;
}

#supprimer{
 color: #b1b1b1;
    background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #565656, stop: 0.1 #525252, stop: 0.5 #4e4e4e, stop: 0.9 #4a4a4a, stop: 1 #464646);
    border-width: 1px;
    border-color: #1e1e1e;
    border-style: solid;
    border-radius: 30px;
    padding: 3px;
    font-size: 12px;
    padding-left: 5px;
    padding-right: 5px;
}

#ajouter:pressed
{
    background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #2d2d2d, stop: 0.1 #2b2b2b, stop: 0.5 #292929, stop: 0.9 #282828, stop: 1 #252525);
}

#modifier:pressed
{
    background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #2d2d2d, stop: 0.1 #2b2b2b, stop: 0.5 #292929, stop: 0.9 #282828, stop: 1 #252525);
}

#supprimer:pressed
{
    background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #2d2d2d, stop: 0.1 #2b2b2b, stop: 0.5 #292929, stop: 0.9 #282828, stop: 1 #252525);
}*/
#pdf_3{
background-color: #071e26;
border:0px ;
}

#stat_2{
	
	background-color: #071e26;
border:0px ;
}

#widget QLineEdit{
background-color: rgb(204, 204, 204);
	border:1px solid rgb(0,0,0);
	
}</string>
  </property>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>140</y>
     <width>1631</width>
     <height>821</height>
    </rect>
   </property>
   <widget class="QLabel" name="form">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>451</width>
      <height>661</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>130</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>nom </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_6">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>240</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>prénom</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_8">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>430</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>num CIN</string>
    </property>
   </widget>
   <widget class="QLabel" name="options">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>60</y>
      <width>181</width>
      <height>661</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="supprimer">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>320</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>supprimer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="modifier">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>220</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>modifier</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ajouter">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>130</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>ajouter</string>
    </property>
    <property name="autoRepeatDelay">
     <number>300</number>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>70</y>
      <width>771</width>
      <height>651</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-raduis:5px;</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>80</y>
      <width>731</width>
      <height>91</height>
     </rect>
    </property>
    <widget class="QLineEdit" name="recherche_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>671</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>  recherche</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>341</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>ASC</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>DESC</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBox_4">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>50</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>NOM</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PRENOM</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CIN</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QLineEdit" name="lineEdit_7">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>530</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>numero télephone</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>180</y>
      <width>771</width>
      <height>541</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QTableView" name="tab_client">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>0</y>
        <width>761</width>
        <height>511</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QLineEdit" name="id_sup">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>60</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>55</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>saisir cin</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>105</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>PDF</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_33">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>110</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="supprimer_2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>420</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0,0,0);
    background-color: rgb(185, 185, 185);
    border-width: 1px;
    border-color: #1e1e1e;
    border-style: solid;
    border-radius: 30px;
    padding: 3px;
    font-size: 12px;
    padding-left: 5px;
    padding-right: 5px;</string>
    </property>
    <property name="text">
     <string>afficher</string>
    </property>
   </widget>
   <widget class="QPushButton" name="imp">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>510</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0,0,0);
    background-color: rgb(185, 185, 185);
    border-width: 1px;
    border-color: #1e1e1e;
    border-style: solid;
    border-radius: 30px;
    padding: 3px;
    font-size: 12px;
    padding-left: 5px;
    padding-right: 5px;</string>
    </property>
    <property name="text">
     <string>PDF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="imp_2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>600</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0,0,0);
    background-color: rgb(185, 185, 185);
    border-width: 1px;
    border-color: #1e1e1e;
    border-style: solid;
    border-radius: 30px;
    padding: 3px;
    font-size: 12px;
    padding-left: 5px;
    padding-right: 5px;</string>
    </property>
    <property name="text">
     <string>trie</string>
    </property>
   </widget>
   <widget class="QComboBox" name="lineEdit_5">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>350</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>tunis</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Autre</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ben arouse</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Manouba</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ariana</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1671</width>
      <height>821</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="style.qrc">:/new/prefix1/icons/blue2.png</pixmap>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>form</zorder>
   <zorder>lineEdit_3</zorder>
   <zorder>lineEdit_6</zorder>
   <zorder>lineEdit_8</zorder>
   <zorder>options</zorder>
   <zorder>supprimer</zorder>
   <zorder>modifier</zorder>
   <zorder>ajouter</zorder>
   <zorder>tableView</zorder>
   <zorder>widget</zorder>
   <zorder>lineEdit_7</zorder>
   <zorder>tabWidget</zorder>
   <zorder>supprimer_2</zorder>
   <zorder>imp</zorder>
   <zorder>imp_2</zorder>
   <zorder>lineEdit_5</zorder>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>2</width>
     <height>2</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QWidget" name="side_menu_2" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>181</width>
     <height>831</height>
    </rect>
   </property>
   <widget class="QPushButton" name="pushButton_16">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Equipements</string>
    </property>
    <property name="icon">
     <iconset resource="style.qrc">
      <normaloff>:/new/prefix1/icons/equi.png</normaloff>:/new/prefix1/icons/equi.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_18">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>         Clients</string>
    </property>
    <property name="icon">
     <iconset resource="style.qrc">
      <normaloff>:/new/prefix1/icons/cli.png</normaloff>:/new/prefix1/icons/cli.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_21">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>        Vehicules</string>
    </property>
    <property name="icon">
     <iconset resource="style.qrc">
      <normaloff>:/new/prefix1/icons/vehi.png</normaloff>:/new/prefix1/icons/vehi.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>45</width>
      <height>45</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="top_widget" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>1831</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#top_widget
{
border-radius:20px;
}

QPushButton 
{
border-radius: 5px
}


</string>
   </property>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>1740</x>
      <y>30</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="style.qrc">
      <normaloff>:/new/prefix1/icons/cli.png</normaloff>:/new/prefix1/icons/cli.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="profil">
    <property name="geometry">
     <rect>
      <x>1570</x>
      <y>30</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 14pt &quot;Monotype Corsiva&quot;;</string>
    </property>
    <property name="text">
     <string>votre copmte</string>
    </property>
    <property name="icon">
     <iconset resource="style.qrc">
      <normaloff>:/new/prefix1/icons/emp.png</normaloff>:/new/prefix1/icons/emp.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>45</width>
      <height>45</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="facture">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 14pt &quot;Monotype Corsiva&quot;;</string>
    </property>
    <property name="text">
     <string>       facture</string>
    </property>
    <property name="icon">
     <iconset resource="style.qrc">
      <normaloff>:/new/prefix1/icons/facturee.png</normaloff>:/new/prefix1/icons/facturee.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>35</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="facture_2">
    <property name="geometry">
     <rect>
      <x>1410</x>
      <y>30</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 14pt &quot;Monotype Corsiva&quot;;</string>
    </property>
    <property name="text">
     <string>  historique</string>
    </property>
    <property name="icon">
     <iconset resource="style.qrc">
      <normaloff>:/new/prefix1/icons/his.png</normaloff>:/new/prefix1/icons/his.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="facture_3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 14pt &quot;Monotype Corsiva&quot;;</string>
    </property>
    <property name="text">
     <string>statiqtiques</string>
    </property>
    <property name="icon">
     <iconset resource="style.qrc">
      <normaloff>:/new/prefix1/icons/stat.png</normaloff>:/new/prefix1/icons/stat.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>20</y>
      <width>481</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 16pt &quot;MV Boli&quot;;
border-radius:20px;</string>
    </property>
    <property name="text">
     <string>gestion des clients</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>20</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 127);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="style.qrc">
      <normaloff>:/new/prefix1/icons/cli.png</normaloff>:/new/prefix1/icons/cli.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="style.qrc"/>
 </resources>
 <connections/>
</ui>
